### ISSUE TITLE
Optional Disable Shields

### ISSUE THREAD

**Issue by Geoffry-Skionfinschii** [Hampfh/Datapack_SurvivalGames#6]
Implement a sign for shield settings so that shields can be disabled per game

Would it be better to remove the shield and give back the items, or just outright warn the player not to make a shield in that game?

**Comment by Hampfh** [comment]
I guess that could be a solution yes

**Comment by Hampfh** [comment]
From experience warning players usually don't work very well, players will & do ignore such warnings. However, the clearing strategy could work if a system was implemented to give back the items

**Comment by Geoffry-Skionfinschii** [comment]
Hmm i'll experiment with taking and giving back items, the ignoring warnings makes a lot of sense ;-;

**Comment by Geoffry-Skionfinschii** [comment]
#8 I've implemented a working shield clearing, just doesn't have a settings sign

**Comment by Hampfh** [comment]
Your pull request works great, I like your approach. The system is now integrated into the lobby in 5fd0330. One thing I did notice was that I had errors and couldn't run the script at first, after some bugtesting I found that the variable `Trig_AllowShields` is more than 16 characters which is not allowed by Minecraft so I renamed it to `Trig_AllowShield` & `AllowShield` for consistency reasons. Tho that was the only problem I had, the script itself works like a charm.
